foundry.toml

```diff
@@ -30,4 +30,10 @@ mainnet = "${ETHEREUM_RPC_URL}"
 goerli = "${GOERLI_RPC_URL}"
 polygon = "${POLYGON_RPC_URL}"
 
+
+[invariant]
+runs = 100
+depth =1
```

Comment:
runs is the number of times a randomized call sequence will be generated
depth is the number of random calls that will be made

This was set to 1 to debug a handler function but we should put it back to its default of 15 or so when ready